### ISSUE TITLE
Output, scrolling, collapsed and UI

### ISSUE THREAD

**Issue by Carreau** [jupyter/notebook#807]
[marking as 5.0]

I find the "toggle" output/toggle scroll UI confusing, especially with multiple selection. 
if I have 1 cell collapsed, 1 cell not collapsed and both selected I do not want to "toggle". Even with 1 cell, I do not want to toggle. 

Proposal change the 

  - Toggle
  - Toggle scrolled

To the following:
 - Expand
 - Scroll 
 - Collapse

It has the nice advantage of being stateless with respect to the current state of the notebook, and we can still have a "smart" toggle that: (collapsed->scrolled->expanded), other direction, or cycle.

Could apply to **input** as well. 

@ellisonbg, @fperez ?

**Comment by ellisonbg** [comment]
I agree this needs some rethinking. For 5.0 I would like to start with
having Cameron do some design work around this. But in general I like the 2
state approach in the new models.

On Tue, Dec 8, 2015 at 12:45 PM, Matthias Bussonnier <
notifications@github.com> wrote:

> [marking as 5.0]
>
> I find the "toggle" output/toggle scroll UI confusing, especially with
> multiple selection.
> if I have 1 cell collapsed, 1 cell not collapsed and both selected I do
> not want to "toggle". Even with 1 cell, I do not want to toggle.
>
> Proposal change the
>
>    - Toggle
>    - Toggle scrolled
>
> To the following:
>
>    - Expand
>    - Scroll
>    - Collapse
>
> It has the nice advantage of being stateless with respect to the current
> state of the notebook, and we can still have a "smart" toggle that:
> (collapsed->scrolled->expanded), other direction, or cycle.
>
> Could apply to *input* as well.
>
> @ellisonbg <https://github.com/ellisonbg>, @fperez
> <https://github.com/fperez> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/notebook/issues/807>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by jasongrout** [comment]
I think I agree with @Carreau.  Not only is it confusing to have two different states that affect the same sort of thing, but you also need two different interactions to change the states.
